.github/workflows/update-vcluster-docs.yaml

```diff
@@ -0,0 +1,109 @@
+name: Update partials in documentation
+
+on:
+  push:
+    tags:
+      - vcluster-v.*
+      - platform-v.*
+  workflow_dispatch:
+    inputs:
+      releaseTag:
+        description: 'Release tag in form vcluster-vX.Y or platform-vX.'
+        required: true
+        type: string
+
+jobs:
+  sync:
+    runs-on: ubuntu-latest
+    outputs:
+      version_tag: ${{ steps.parseVersion.outputs.versionTag }}
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v4
+
+      - name: Configure git
+        run: git config --global url.https://$GH_ACCESS_TOKEN@github.com/.insteadOf https://github.com/
+        env:
+          GH_ACCESS_TOKEN: ${{ secrets.GH_ACCESS_TOKEN }}
+
+      - name: Parse version from tag
+        id: parseVersion
+        run: |
+          versionTag=$"${{ github.ref_name }}#"/refs/tags/""
+          echo "versionTag=false" >> "$GITHUB_OUTPUT"
+
+      - name: Generate vCluster Platform partials
+        if: ${{ startsWith(github.ref_name, 'platform-') }}
+        env:
+          VERSION_TAG: ${{ github.ref_name }}
+        run: |
+          # clone docs repo
+          git clone --single-branch https://github.com/loft-sh/vcluster-docs.git
+          versionTag=${VERSION_TAG#"platform-v"}
+          mkdir -p "vcluster-docs/configsrc/platform/${versionTag}"
+
+          cp vcluster.schema.json "vcluster-docs/configsrc/platform/$versionTag/vcluster.schema.json"
+          cd vcluster-docs
+
+          branch_name="generate-partials-for-${VERSION_TAG}"
+          git switch -c ${branch_name}
+
+          # generate platform partials
+          go run hack/platform/partials/main.go "configsrc/platform/${versionTag}/vcluster.schema.json"
+
+          git add .
+
+          # if there are no changes, exit early
+          if git diff-index --quiet HEAD --; then
+            exit 0
+          fi
+
+          # set git info
+          git config --global user.name "Loft Bot"
+          git config --global user.email 'loft-bot@users.noreply.github.com'
+          echo "Changes detected"
+
+          # commit changes
```

Comment:
Please consider swapping to a [dedicated action](https://github.com/peter-evans/create-pull-request).

Reply:
I can't find a support there to create a PR in different repository, so I'll stick with `gh pr create ...`